### PR TITLE
Update DWARF testcases

### PR DESCRIPTION
**Pull request by kripken**
This only touches test code. The files are compiled with
latest LLVM + https://reviews.llvm.org/D71681 in order
to get more realistic DWARF content.

Not much of interest here, except for
```
DW_AT_frame_base [DW_FORM_exprloc]	(<decoding error> 01 00 9f)
```
which perhaps means I should update the LLVM in
binaryen, is that one of the new changes?

### CODE REVIEW

**Comment by dschuff** [comment]
Yes, the frame base is encoded as a wasm local. You'll need at least https://github.com/llvm/llvm-project/commit/adf7a0a558a51f275bf8906b6a010c397560b7ff .

File: test/passes/fannkuch3_manyopts.bin.txt [hunk 1]
```diff
@@ -2265,622 +2368,760 @@ Abbrev table for offset: 0x00000000
...
-                  DW_AT_type [DW_FORM_ref4]	(cu + 0x0045 => {0x00000045} "int")
+                  DW_AT_type [DW_FORM_ref4]	(cu + 0x0059 => {0x00000059} "int")
 
-0x000000ee:     DW_TAG_variable [18]  
+0x000000c3:     DW_TAG_variable [13]  
                   DW_AT_location [DW_FORM_sec_offset]	(0x0000001d: 
-                     [0xffffffff,  0x00000003): 
+                     [0xffffffff,  0x00000006): 
                      [0x00000007,  0x0000003c): DW_OP_consts +0, DW_OP_stack_value
+                     [0x00000052,  0x00000057): <decoding error> 00 01 9f
```
Comment (dschuff) [inline test/passes/fannkuch3_manyopts.bin.txt]: this is also expected, as it comes from a llvm.dbg.value intrinsic and should also encode a wasm local.

**Comment by kripken** [comment]
Hmm, adding that llvm commit in #2596 does not fix things in my testing. Any obvious reason why? I'll debug...

**Comment by kripken** [comment]
Oh nv, I missed part of the patch...

**Pull request merged**